assets-src/css/overrides.css

```diff
@@ -22,19 +22,4 @@
 .article-wrapper img {
     max-width: 100%;
     height: auto;
-}
-
-.home .card {
```

Comment:
@sviluppoRobyone Mi sembra che la modifica dell'altezza delle card fosse stata fatta per risolvere un problema di visualizzazione contenuti nel caso ci fosse più testo del previsto. Dovrei ritrovare la issue...

`min-height` mi torna di più, `max-height` non tanto invece.

Reply:
@sviluppoRobyone questa era la issue che era stata risolta con queste altezze fisse https://github.com/italia/design-scuole-wordpress-theme/issues/302

Non sono state introdotte nella v2.12.2

Forse si potrebbero gestire con Flexbox?

Reply:
se puoi, scorporiamo il fix sui colori delle sezioni da quello dell'altezza delle card che merita un approfondimento. teniamo aperta questa PR per i colori, così procediamo con il merge e rilascio nella prossima, imminente versione. Grazie

Reply:
Ci siamo permessi di approfondire e ricostruire la cronistoria.

L'altezza fissa è stata introdotta nel commit che risolve la issue #302. Tuttavia, in un commit successivo, era stata rimossa (commit [8dfa26e](https://github.com/italia/design-scuole-wordpress-theme/commit/8dfa26e98eb550c1344ee7a2cedf7e6aec7d42e0#diff-e8eff0975d30ae4013561b11a589489938009a4fda06462ab26128b2d38761f9)).
Infine, nell'ultima release, il file overrides è stato ricompilato, annullando inaspettatamente la rimozione fatta in precedenza.

Il nostro commit va quindi a ripristinare la situazione corretta.

Ci teniamo a ribadirlo, perché attualmente il problema con i testi persiste e ci sembra strano rallentare questa correzione, contando che per quanto ci riguarda la PR può rimanere aperta finché effettuate gli approfondimenti del caso.